docs/tips_and_tricks.md

```diff
@@ -0,0 +1,13 @@
+# Tips & Tricks
+
+## Synchronizing containers time with host time
+
+If you want to synchronize containers time with host time they
```

Comment:
containers' time with the time of the host they

Reply:
Done